vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
This file (all of vendor) should be gitignored.

I see it's present In the gitignore but has been added to the repo beforehand so the commands on here should sort it for you

https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore